docs/platform/howto/use-aws-privatelinks.rst

```diff
@@ -142,25 +142,63 @@ currently support AWS PrivateLink.
    successful heartbeats before they transition from the ``initial``
    state to ``healthy`` and are included in the active forwarding rules of the load balancer.
 
-   | **Note:** Currently, you can only create one VPC endpoint for each
-     Aiven service.
-
 .. _h_b6605132ff:
 
-Connection information
-----------------------
+Acquire connection information
+------------------------------
+
+.. _one-connection:
+
+One AWS PrivateLink connection
+''''''''''''''''''''''''''''''
+
+Once you have enabled AWS PrivateLink access for a service component, the toggle switch for the ``privatelink`` access route is available in the **Connection information** section on the **Overview** page in `Aiven Console <https://console.aiven.io>`__. Values for ``host`` and ``port`` (for some service components, such as Kafka) differ from those for the default ``dynamic`` access route used to connect to the service.
+
+.. note::
+
+   You can use the same credentials with any access route.
+
+Multiple AWS PrivateLink connections
+''''''''''''''''''''''''''''''''''''
+
+If you have more than one AWS PrivateLink connection, you can get connection information for the first endpoint as described in :ref:`One AWS PrivateLink connection <one-connection>` (from `Aiven Console <https://console.aiven.io>`__). For connection information for the remaining endpoints, you need to use CLI.
+
+Each endpoint (connection) has PRIVATELINK_CONNECTION_ID, which you can check using the :doc:`avn service privatelink aws connection list SERVICE_NAME </docs/tools/cli/service/privatelink>` command.
+
+To acquire connection information for your service component using AWS PrivateLink, run the :doc:`avn service connection-info </docs/tools/cli/service/connection-info>` command.
+
+* For SSL connection information for your service component using AWS PrivateLink, run the following command:
+
+.. code-block:: bash
+
+   avn service connection-info UTILITY_NAME SERVICE_NAME -p PRIVATELINK_CONNECTION_ID
+
+.. topic:: Where
+
+  * UTILITY_NAME is ``kcat``, for example
+  * SERVICE_NAME is ``kafka-12a3b4c5``, for example
+  * PRIVATELINK_CONNECTION_ID is ``plc39413abcdef``, for example
+
+* For connection information for your service component using AWS PrivateLink with SASL enabled, run the following command:
+
+.. code-block:: bash
+
+   avn service connection-info UTILITY_NAME SERVICE_NAME -p PRIVATELINK_CONNECTION_ID -a sasl
+
+.. topic:: Where
+
+  * UTILITY_NAME is ``kcat``, for example
+  * SERVICE_NAME is ``kafka-12a3b4c5``, for example
+  * PRIVATELINK_CONNECTION_ID is ``plc39413abcdef``, for example
+
```

Comment:
These are Kafka specific, may be have them rather as an example for Kafka? they are pretty useful though so I won't omit them either.